README.md

```diff
@@ -95,9 +95,22 @@ A single directory is mounted at `/job` in each container before it is run, and
 
 This helps facilitate cross-stage communication, which becomes particularly useful if certain stages need to pass along results.
 
+### Private Registry Support
+
+If you choose to [set up a private registry](https://docs.docker.com/registry/deploying/), you should secure it. After it is secured, set up your credentials in the docker daemon using 
```

Comment:
You think it would be worth embedding "secure it" with the link https://docs.docker.com/registry/deploying/#run-an-externally-accessible-registry?

Reply:
I'm not sure. That is on the page that is already linked. It might be a bit overwhelming to link every part of that sentence.